src/main/java/org/repodriller/persistence/csv/CSVFile.java

```diff
@@ -149,8 +150,10 @@ public synchronized void write(Object... fields) throws CSVFileFormatException {
 
 	@Override
 	public synchronized void close() {
-		ps.close();
-		writable = false;
+		if (isOpen) {
```

Comment:
isn't it the same as checking `== null` ?

If `open()` fails, then `ps` will be null. Not sure if we need this auxiliary variable

Reply:
Trying to protect against multiple close() calls on a CSVFile. I'm not sure if the underlying PrintStream class supports multiple close-s, the docs are silent on the matter.

Could also be addressed by explicitly setting `ps` to null after closing it.

Reply:
I'll merge it because it definitely solves the problem.

Right now, as `PersistenceMechanism` are closed by Repodriller, I don't see this as a big issue. But if we change this in the API, we can revisit this implementation.